hadoop-common-project/hadoop-minikdc/src/test/java/org/apache/hadoop/minikdc/TestMiniKdc.java

```diff
@@ -38,8 +38,19 @@
 import java.util.Arrays;
 
 public class TestMiniKdc extends KerberosSecurityTestcase {
-  private static final boolean IBM_JAVA = System.getProperty("java.vendor")
-      .contains("IBM");
+  private static final boolean USE_IBM_JAVA_PACKAGES = shouldUseIbmPackages();
```

Comment:
should just use PlatformNames

Reply:
Thanks again, I have implemented the change in [5eb229d](https://github.com/apache/hadoop/pull/4537/commits/5eb229d746e25c54f456da06419267ff6222d541) which required me to add the auth module as a test scoped dependency which was the initial reason I avoided doing this. It seemed as if there may have been a good reason not to based on the existing implementation.